packages/sw-cli/src/index.js

```diff
@@ -119,9 +120,8 @@ class SWCli {
     switch (command) {
       case 'generate-sw':
         return this._generateSW();
-      case 'build-file-manifest':
-        logHelper.error(`TODO: Implement.`);
-        return Promise.reject();
+      case 'generate-manifest':
```

Comment:
Sorry for the late comment. For the initial MVP of sw-cli, I think opting for the generate-* style commands makes sense. Longer term we might want to consider whether we'd benefit from grouping these the way we do in Yeoman: e.g `sw-cli generate:sw` `sw-cli generate:manifest` etc. 

We might also want to think about whether we'll ever need to support method chaining for these commands, but that can come later.